Assets/EnableFinalBattle.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnableFinalBattle : MonoBehaviour

```

Comment:
I think `EnableFinalBattle` as a verb isn't a good idea for a class. 

Have you considered extending the CombatZone class? I feel like FinalBattleZone is a perfect candidate for  `is a combat zone`.

Reply:
I'll probably refactor this later haha